test/sideinput-e2e/sideinput-e2e_sink_source_test.go

```diff
@@ -0,0 +1,54 @@
+//go:build test
+
+/*
+Copyright 2022 The Numaproj Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package sideinput_e2e
+
+import (
+	. "github.com/numaproj/numaflow/test/fixtures"
+	"github.com/stretchr/testify/suite"
+	"testing"
+	"time"
+)
+
+type SideInputUDSSuite struct {
+	E2ESuite
+}
+
+func (s *SideInputUDSSuite) setUpTests(pipeLineFile string) *When {
+	w := s.Given().Pipeline(pipeLineFile).When().CreatePipelineAndWait()
+	w.Expect().VertexPodsRunning()
+	return w
+}
+
+func (s *SideInputUDSSuite) TestSinkWithSideInput() {
+	w := s.setUpTests("@testdata/sideinput_sink.yaml")
+	defer w.DeletePipelineAndWait()
+	w.Expect().SinkContains("redis-uds", "e2e-even", WithTimeout(5*time.Minute))
```

Comment:
I think 5min timeout is too long for e2e, do we need to wait for this long? or it can be reduced to probably 1min?